packages/ui/src/proposals/model/proposalDescriptions.ts

```diff
@@ -37,6 +37,7 @@ export const proposalDescriptions: ProposalDescriptions = {
     'Proposal to submit the Channel Incentives Payout payload, update the min/max claimable amounts and block/allow claiming of awarded JOYs by channels.',
   runtimeUpgrade: 'Proposal to upgrade version to the new runtime.',
   updatePalletFrozenStatus: 'Enabling and disabling various features on the blockchain level.',
+  setEraPayoutDampingFactor: 'TODO',
```

Comment:
Secondly could you point me somewhere I could get a clear explanation of how will the era payout damping factor affect the validator payouts. I need this to write this short description of the proposal and I might also add a longer description to the `SetEraPayoutDampingFactor` component.